lib/regular/regular.mli

```diff
@@ -580,7 +580,7 @@ module Std : sig
         ?dump  : (Out_channel.t -> 'a -> unit) ->
         ?pp    : (Format.formatter -> 'a -> unit) ->
         ?size  : ('a -> int) ->
-        ?blit_to_string:('a,string) copy ->
+        ?blit_to_bytes:('a,bytes) copy ->
```

Comment:
we need to keep `blit_to_string` for compatibility, though adding `blit_to_bytes` is also a good idea.